mailpoet/assets/js/src/common/sender-domain-notice/sender-domain-notice-body.tsx

```diff
@@ -0,0 +1,62 @@
+import { createInterpolateElement } from '@wordpress/element';
+import { escapeHTML } from '@wordpress/escape-html';
+import { __ } from '@wordpress/i18n';
+
+function SenderDomainNoticeBody({
+  emailAddressDomain,
+  isFreeDomain,
+  isPartiallyVerifiedDomain,
+  isSmallSender,
+  onlyShowWarnings = false,
+}: {
+  emailAddressDomain: string;
+  isFreeDomain: boolean;
+  isPartiallyVerifiedDomain: boolean;
+  isSmallSender: boolean;
+  onlyShowWarnings?: boolean;
+}) {
+  const renderMessage = (messageKey: string) => {
+    const messages: { [key: string]: string } = {
+      freeSmall:
+        "Shared 3rd-party domains like <emailDomain/> will send from MailPoet's shared domain. We recommend you to use your site's branded domain instead.",
```

Comment:
The wording is just a little awkward to my ear. I would recommend this small change:
```suggestion
        "Shared 3rd-party domains like <emailDomain/> will send from MailPoet's shared domain. We recommend that you use your site's branded domain instead.",
```

Reply:
Changed